org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.domainmodel/build.properties

```diff
@@ -12,4 +12,7 @@ additional.bundles = org.eclipse.xtext.xbase,\
                      org.eclipse.xtext.xtext.generator,\
                      org.eclipse.emf.codegen.ecore,\
                      org.eclipse.emf.mwe.utils,\
-                     org.eclipse.emf.mwe2.launch
+                     org.eclipse.emf.mwe2.launch,\
+                     org.apache.log4j,\
+                     org.apache.commons.logging,\
+                     com.ibm.icu
```

Comment:
The compile error I mentioned was a phantom one. It did not appear with a fresh project.